### DISCUSSION TITLE
The complete exception is provided below:

### DISCUSSION THREAD

**Discussion by SerhiiZahuba** [mattieserver/netbox-topology-views#534] [Q&A]
### NetBox version

4.0.

### Topology Views version

4.0.0

### Steps to Reproduce

1) Set filters
2) Click Search

### Expected Behavior

Result

### Observed Behavior

Error 500

![image](https://github.com/netbox-community/netbox-topology-views/assets/36307887/4d90e415-4ae2-4ef2-b9c4-d53d36430cfe)
![image](https://github.com/netbox-community/netbox-topology-views/assets/36307887/9cda1919-b019-4545-99d6-e3971d44a4e4)

**Comment by mattieserver** [comment]
Unfortunately i do not have enough info to help you.

Please provide all the filters you used (the URL is fine, without the domain).

Also try with different filters to try and see if you can find if the issue is linked to a specify device.

**Comment by SerhiiZahuba** [comment]
I've tried different options for devices\filters but I keep getting the error.
Maybe i can get a debug of this somewhere?

**Comment by dreng** [comment]
> I've tried different options for devices\filters but I keep getting the error. Maybe i can get a debug of this somewhere?

If you need help you should follow the advise:

> Please provide all the filters you used (the URL is fine, without the domain).

**Comment by SerhiiZahuba** [comment]
Hello. This is my url

https://netbox.**.com/plugins/netbox_topology_views/topology/?q=&group=&show_unconnected=&show_cables=True&show_logical_connections=True&show_single_cable_logical_conns=&show_neighbors=True&show_circuit=&show_power=&show_wireless=&group_sites=&group_locations=&group_racks=&group_virtualchassis=&straight_cables=&grid_size=&site_id=43&site_id=22&site_id=44&site_id=8&serial=&asset_tag=&mac_address=&console_ports=&console_server_ports=&power_ports=&power_outlets=&interfaces=&pass_through_ports=&has_primary_ip=&has_oob_ip=&virtual_chassis_member=&local_context_data=&cf_DataCenterID=&cf_vcsa_host_cpu_cores=&cf_vcsa_host_memory=
